app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -256,8 +284,16 @@ export const FormAudience = ({
             <Form.Check
               {...formControlAttrs("isSticky")}
               type="checkbox"
+              checked={isSticky}
+              onChange={(e) => setIsSticky(e.target.checked)}
+              disabled={stickyRequiredWarning}
               label="Sticky Enrollment (Clients remain enrolled until the experiment ends)"
             />
+            {stickyRequiredWarning && (
+              <FormLabel data-testid="stickyRequiredWarning">
+                Sticky enrollment is required for this targeting configuration.
```

Comment:
Let's use `.text-warning` for this.

Reply:
Sorry actually no, not text warning, a [warning alert](https://react-bootstrap.github.io/components/alerts/#examples).